src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -115,6 +115,7 @@ def _complex_params(command_group):
 
         with self.argument_context('{} server configuration set'.format(command_group)) as c:
             c.argument('value', help='Value of the configuration. If not provided, configuration value will be set to default.', validator=configuration_value_validator)
+            c.argument('configuration_name', options_list=['--name', '-n'], help='The name of the configuration')
```

Comment:
Why remove `id_part='child_name_1'` here? This will disable customers using resource id...